lib/travis/build/appliances/services.rb

```diff
@@ -24,6 +24,15 @@ def apply
               end
             end
 
+            if services.delete('mysql')
+              sh.if "$(grep -E '^socket = /run/mysql-5.6' #{HOME_DIR}/.my.cnf)" do
+                sh.cmd "sudo service mysql-5.6 start", assert: false, echo: true, timing: true
```

Comment:
I suspect a service alias is the easiest solution.  I don't want to take on the burden of maintaining our own MySQL cookbook (again), and the community cookbook we're using adds a suffix to `mysql-` by default.